src/channel.rs

```diff
@@ -884,9 +884,9 @@ impl Channel {
     ///
     /// fn main()
     /// {
-    ///     let url = "https://feedpress.me/usererror.xml";
+    ///     let input = include_str!("tests/data/rss2sample.xml");
     ///
-    ///     Channel::from_url(url).unwrap();
+    ///     let channel = input.parse::<Channel>().unwrap();
```

Comment:
This change should be reverted since this doc is describing the `from_url` method